monitor.go

```diff
@@ -49,12 +49,23 @@ func (m *Monitor) serve() error {
 	return nil
 }
 
+// Actually is a double new line
+var newLine = []byte{'\r', '\n', '\r', '\n'}
+
 func (m *Monitor) handleRaw(addr net.Addr, raw []byte) error {
+	// Add newline to workaround buggy SSDP responses
+	if !bytes.HasSuffix(raw, newLine) {
+		raw = bytes.Join([][]byte{raw, newLine}, nil)
+	}
 	if bytes.HasPrefix(raw, []byte("M-SEARCH ")) {
 		return m.handleSearch(addr, raw)
 	}
 	if bytes.HasPrefix(raw, []byte("NOTIFY ")) {
-		return m.handleNotify(addr, raw)
+		err := m.handleNotify(addr, raw)
+		if err != nil {
+			logf("error parsing notify: %s", err)
```

Comment:
is this log absolutely necessary?

i want to remove this.

Reply:
I added this because I didn't have any log telling me that the NOTIFY was erroneous. It was indeed getting and "Unexpected EOF" from handleNotify which was not logged.